CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ on GitHub.
 
 ## 3.4.0-SNAPSHOT
 
+**This release officially introduces Spring Boot 2.7 compatibility.**
+Note that the previous releases of Spring Cloud GCP 3.x are also compatible with Spring Boot 2.7. The one exception is that if you use Cloud SQL with R2DBC, you'd have to manage the driver versions in your own application dependencies.
+
```

Comment:
optional: Do you think it will be helpful to reference the 2.7 part of the `sql.adoc` here?

Reply:
Ah definitely didn't think this through. Makes sense!

Reply:
Decided to made it a reference to source code; seems less confusing. At least that will be available once this PR is merged.